zhazam/Modules/LeaderBoard/LeaderBoardViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  LeaderBoardViewController.swift
+//  zhazam
+//
+//  Created by Abai Kalikov on 5/9/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+final class LeaderBoardViewController: UIViewController, Reusable {
+    
+    private enum Constants {
+        static let tableViewRowHeight: CGFloat = 65
+    }
+    
+    @IBOutlet private var tableView: UITableView!
+    
+    lazy var gameTypeButton: UIBarButtonItem = {
+        let rightBarButtonItem = UIBarButtonItem(title: "Classic",
+                                                 style: .plain,
+                                                 target: self,
+                                                 action: #selector(didTapped))
+        navigationItem.rightBarButtonItem = rightBarButtonItem
+        return rightBarButtonItem
+    }()
+    
+    private let storage: LeaderBoardStorage
+    var leaderBoards = [LeaderBoardRowModel]()
+    
+    init(storage: LeaderBoardStorage) {
```

Comment:
you do not have to initialize your storage in init. Do it instantly. I used the initializer in my classes because there was a variation of different storages. You have only one kind of storage here&

Reply:
Okay, got it